pkg/aws/helpers.go

```diff
@@ -31,7 +31,7 @@ import (
 var RoleNameRE = regexp.MustCompile(`^[\w+=,.@-]+$`)
 
 var AuditLogArnRE = regexp.MustCompile(
-	`^arn:aws:iam::\d{12}:role/[a-zA-Z0-9_-]+$`,
+	`^arn:aws:iam::\d{12}:role(?:\/[\w+=,.@-]+)*(?:\/[a-zA-Z0-9_-]+)$`,
```

Comment:
Can we document what this is exactly checking. Ideally a link to a spec or schema would be great. Just for the next engineer.